scripts/install2

```diff
@@ -507,6 +508,26 @@ fi
 sudo cp status /usr/local/bin/status
 echo -e "Run \e[1;32mstatus\e[0m anytime to print the status of your node"
 
+# Setup nodeinfo.json, adding non-static info
+echo '"installed":'"$(date)"', "uri":"https://github.com/tomeshnet/prototype-cjdns-pi/tree/develop"},' >> shared/nodeinfo/nodeinfo.json
```

Comment:
Why are we modifying local files  on install?

Reply:
To add dynamic things to the json file, like node ids and things. I realize now that just putting it in the nodeinfo install and running that is better.